client/electron/src/index.ts

```diff
@@ -50,6 +50,29 @@ if (electronIsDev) {
   // autoUpdater.checkForUpdatesAndNotify();
 })();
 
+const lock = app.requestSingleInstanceLock();
+
+if (!lock) {
+  app.quit();
+} else {
+  app.on('second-instance', (_event, commandLine, _workingDirectory) => {
+    if (!myCapacitorApp.isMainWindowVisible()) {
+      myCapacitorApp.showMainWindow();
+    } else {
+      myCapacitorApp.getMainWindow().focus();
+    }
+
+    if (commandLine.length > 0) {
+      const lastArg = commandLine.at(-1);
+      // We're only interested in potential Parsec links
+      if (lastArg.startsWith('parsec://')) {
+        myCapacitorApp.getMainWindow().webContents.send('open-link', lastArg);
+      }
```

Comment:
Electron provide a command line parser, I recommend to use it since an user can provide additional flag to tweak the electron behavior

https://www.electronjs.org/docs/latest/api/command-line-switches

Reply:
I don't see anything in this doc that is any help, at least not if we use the same format as V2 (where the link is given as a solo argument). Also keep in mind that this will never be called directly by anyone (outside of testing), it will be called by the `parsec://` protocol registration.

Reply:
@Max-7 conclusion looks goot to me atm.